meetings/2021-06-09.md

```diff
@@ -0,0 +1,137 @@
+* Justin B
+* Matthieu B
+
+
+## Minutes
+
+Henry: Currently we have access control rules and inheritance that works with default.
+What if you want to add someone to only one of the weekly-meetings?
+Then you currently have to copy all the rules.
+
+Matthieu: Yes, let's elaborate with that.
+
+Henry: I think there is part of the use case where you add someone, we should just use that and this is a classic case of DRY.
+
+Justin: If you have different sets of agents and try to maintain that, assume it's gona break.
+
+Henry: It will become easier to pinpoint and show what the problem is.
+
+Matthieu: We could maybe do imports via link headers.
+
+Henry: I have a default ACL on the filesystem and you create a new acl with each resource and import its parent acl.
+That's what I show in [210 if you do a query asking for trig](https://github.com/solid/authorization-panel/issues/210#issuecomment-838747077).
+If you get trig, you might as well have all the relevant info.
+
+Matthieu: What if you don't have access to all the access control rules?
+This is why I was suggesting link headers of rel type `owl:import`.
+
+### UMA
+
+Justin: I think we made really good traction on WAC & ACP. We have a draft of ACP that should be able to be reviewed soon. I wanted to ask for convergence on the workflows required for interoperability and consent.
+The pattern is "trusted agent" - "authorization server" the trusted authorization agent is trusted to give authorization.
+UMA is a pattern that was related to interop workflows that is pretty OIDC centric.
+This should be the forum in which we start cooperating and looking at.
+It would be a pretty in depth work track compared to the WAC/ACP track.
+It would cover how applications spec what they need to an authorization agent, how the agent presents it, how the authorization is made, how that's stored and persisted.
+There is a vein of it as well about verifiable credentials.
+At a minimum we need to start talking about it to figure out where it's gonna live.
+We rely on this pattern in interoperability and there's gonna be reliance on it in the ecosystem.
+But we haven't dug into the mechanics of how you communicate with your authorization agent.
+It feels like the right time to dig in here (I know Pavlik agrees).
+Maybe even have 
+
+Matthieu: Would UMA be a way to advertise which credentials are required to access a resource.
+
+Justin: UMA existed before VC. It's one pattern we rely on in interop so users can provide fine grained access to their data.
+I'm not proposing starting working on UMA, just looking at the workflow patterns of authorization agents.
+Can you use or take advantage of verifiable credentials in a number of ways?
+Storing decisions. Doing use case for authorization by credentials. I don't know which authorization you want and sending them all defeats the pattern.
+An authorization server you trust would be an answer.
+The pattern of having you authorisation agent would answer some questions.
```

Comment:
I think this would be your own authorization agent.
But it was definitely a more challenging stream of thoughts to accurately capture.
Maybe @justinwb can help fill the blanks ^^,
I'm afraid I lost a bit here.

Reply:
```suggestion
The pattern of having your own authorisation agent would answer some questions.
```